### ISSUE TITLE
False Positive When Checking JSON Translation Files

### ISSUE THREAD

**Issue by pkalusek** [LarsWiegers/laravel-translations-checker#54]
When using Laravel-Langs option to add translations they are saved as json files like

- lang/de.json
- lang/en.json

Wouldn't the second condition in the if result in a false positive then? 
https://github.com/LarsWiegers/laravel-translations-checker/blob/205cb106dd0d96be1a117ca14c75958eb6aaae86/src/Console/Commands/CheckIfTranslationsAreAllThereCommand.php#L234-L239
It seems like the check might find the original file itself in the loop, even though the translation doesn't actually exist in the other languages. This happens because the language distinction is in the filename, not the folder name.

I debugged the code and for example, when processing de.json, it would work like this:

```
$fileName = 'de.json';
$baseDirectory = '.../lang';
```

Then, in the foreach loop when checking other languages like  `$langauge = 'en'`, it checks whether either `.../lang/en/de.json` or `.../lang/de.json` (second condition) exists. Of course, `.../lang/de.json` is the original file being evaluated.
So the logic might incorrectly assume the translation exists for all languages, since it's checking against the same file.

Is this intentional? What case does the second condition cover?
Understanding the purpose might help in finding a good solution for this edge case.
It's not a major issue, since potentially missing translation keys are still noted, but I wanted to point it out just in case.
